src/playback/qml/MuseScore/Playback/internal/MixerTitleSection.qml

```diff
@@ -63,6 +64,21 @@ MixerPanelSection {
         border.color: labelColor
         border.width: 1
 
+        MouseArea{
```

Comment:
Please move it to the end of the rectangle (after StyledTextLabel)
The mouse area is usually located at the end of the container so that no one overlaps it